### DISCUSSION TITLE
New Argon numbers aren't displayed in non-Argon skins

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#25427] [Q&A] [answered]
More specifically, it seems that the new numbers from the Argon skin aren't displayed if you try to create a ArgonAccuracyCounter / ArgonComboCounter / ArgonScoreCounter in a skin that isn't the Argon skin. The score counter doesn't display anything and the accuracy / combo counters only display the text but no actual numbers.

![osu_2023-11-13_22-47-24](https://github.com/ppy/osu/assets/74088315/bba26677-411c-4e56-b423-3968b2ef4b77)

**Comment by Stoppedpuma** [comment, marked as answer]
#25430
